core/tauri/src/scope/fs.rs

```diff
@@ -106,10 +107,29 @@ impl Scope {
       }
     }
 
+    let require_literal_leading_dot = match scope {
+      FsAllowlistScope::Scope {
+        require_literal_leading_dot: Some(require),
+        ..
+      } => *require,
+      // dotfiles are not supposed to be exposed by default on unix
+      #[cfg(unix)]
```

Comment:
Actually reversed behavior and should be true on unix by default.

When `require_literal_leading_dot` is set to `true` it will strip the file paths, which contain `.` in their first path component (eg. `.ssh`).
By setting `require_literal_leading_dot` to false by default it will return the *hidden* folders and files on unix.

Reply:
damn me 😅, I had the logic flipped inside my head the whole time I was working on the PR.